2018-westphal-antpc.tex

```diff
@@ -29,42 +32,51 @@
 
 \begin{document}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Abstract}
+Pyroprocessing does not yet have methods to quantify safeguardability of its waste streams \cite{Borrelli_2017}. This work assesses system parameters that influence separation efficiency and throughput to generate a model of various facility designs. Implementing a generic model allows simulations to determine priority locations for detectors and inspection for the timely notification of material diversion. A material flowsheet is created to reflect the planned facility model along with parameters and waste quantities of interest. To analyze data retrieved from the model, a minimum relative entropy model is proposed and simulations discussed.
```

Comment:
- get rid of the first sentence or move it later. If you keep it somewhere, make the sentence less convoluted by using active voice and simpler words "does not yet have" = "lacks"
- Please hardwrap lines.
- No citations in the abstract. It should stand alone. 
- Abstracts are handled as environments in LaTeX. In this particular style, it'll act just like a normal section, but for other style files, it matters. 

```
\begin{abstract} 
text 
\end{abstract}
```

Reply:
"generate a model of various facility designs" <--- this is a confusing way to phrase it. What you are creating is not a model of various designs but a "customizable pyroprocessing facility archetype for use with the Cyclus framework"

Reply:
Implementing a generic model -> "A generic facility model"

Reply:
Simulations don't determine things. 

"allows simulations to determine" -> "allows simulations to inform"

Reply:
notification -> detection

Reply:
"A material flowsheet is created to reflect the planned facility model along with parameters and waste quantities of interest."  ->>> "This paper describes the Pyre facility archetype design, pyroprocessing flowsheets captured by the model, and simulation capabilities it enables."